configure

```diff
@@ -100,6 +101,40 @@ while [ "$TF_NEED_HDFS" == "" ]; do
   esac
 done
 
+while [ "$TF_NEED_LIBJPEG_TURBO" == "" ]; do
```

Comment:
@jart If you do indeed have an internal build file that builds libjpeg-turbo within Bazel, then that would definitely be a preferable way to go. If you could either share that so I could fix this PR, or substitute your solution with a different commit, that would render this entire PR unnecessary. :) Please keep me posted!